.github/DISCUSSION_TEMPLATE/q-a.yml

```diff
@@ -0,0 +1,51 @@
+title: "[Q&A] "
+labels: ["q&a"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out the form so the community can better assist you!
+  - type: input
+    attributes:
+      label: Python version (`python3 -V`)
+      description: Python version (`python3 -V`)
+      placeholder: "3.8"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: NVFlare version (`python3 -m pip list | grep "nvflare"`)
+      description: NVFlare version (`python3 -m pip list | grep "nvflare"`)
+      placeholder: "2.3.8"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: NVFlare branch (if running examples, please use the corresponding branch, `git branch`)
```

Comment:
Suggest reword to "if running examples, please use the branch that corresponds to the NVFlare version"?

Reply:
thanks, reworded